content/en/docs/usage/ingress/ingress-istio.md

```diff
@@ -0,0 +1,234 @@
+---
+title: "Istio as Ingress"
+weight: 2
+description: "Learn how to use Istio as Ingress with Fission."
+---
+
+Istio is one of the most popular open source service mesh.
+It provides a platform to manage microservices and also facilitates communication between them.
+It offers features for traffic management, security, observability and extensibility.
+In this document, we shall look into how you can use Istio as Ingress with Fission.
+
+## Pre-requisites
+
+To begin with, you need to have Istio installed on your cluster.
+There are multiple options to set up Istio and based on your setup, you can choose one of many ways available to [install Istio](https://istio.io/latest/docs/setup/install/).
+
+To install Fission you can refer to [Fission Installation](/docs/installation).
+Set the `enableIstio` flag as `true` during the setup to enable Istio in Fission
+
+```bash
+helm install --version v1.15.1 --namespace fission \
+  --set serviceType=NodePort,routerServiceType=NodePort \
+  --set enableIstio=true \
+  fission-charts/fission-all
+```
+
+Enable `istio-injection` for fission namespace
+
+```bash
+kubectl label namespace fission istio-injection=enabled
+```
+
+At this point, you have fission installed along with Istio sidecar injection enabled.
+Let us now look into various ways of using Istio with Fission.
+
+For this entire documentation, we are going to use our [Single vs Monolith](https://fission.io/blog/single-or-monolith-serverless-functions-what-should-you-choose/) example.
+You can clone it from our [examples repository](https://github.com/fission/examples/tree/master/python/SinglevsMonolith).
+
+## Configure Istio Gateway
+
+One of the features of Istio is its ability to let you easily control the flow of traffic and API calls between services.
+It provides a lot of options to manage traffic coming in to your cluster.
+While Istio supports Kubernetes Ingress, it offers an **Istio Gateway** that provides more customization and flexibility than Ingress.
+
+Kubernetes ingress is a load balancer that receives incoming traffic.
+However, it doesn't have support for configuring traffic routing.
```

Comment:
It depends on ingress actually. Most of ingress ingress support routing. Like path based routing etc.

Reply:
we can remove this statement IMO